events/member_boosted_server_EVT.js

```diff
@@ -11,14 +11,14 @@ module.exports = {
     DBM.Events.boostedGuild = function boostedGuild(old, recent) {
       if (!Bot.$evts['Member Boosted Server']) return;
       const server = recent.guild;
-      if (!(!old.premiumSince && recent.premiumSince)) return;
```

Comment:
Please revert all the changes in the `mod` function, but perhaps replace this if statement with `if (old.premiumSince || !recent.premiumSince) return;` for legibility?